src/server/config/swagger.json

```diff
@@ -97,42 +97,72 @@
         ]
       }
     },
-    "/users/name": {
+    "/status-box/days-to-harvest/{batchId}": {
       "get": {
-        "tags": ["user name"],
-        "summary": "Get a name for specific user by uid",
-        "description": "",
-        "operationId": "getName",
-        "produces": ["application/json"],
+        "tags": ["status box"],
         "parameters": [
           {
-            "name": "authorization",
-            "in": "header",
-            "description": "Firebase token",
-            "required": true,
-            "type": "string",
-            "default": "Bearer ENTER_FIREBASE_TOKEN"
+            "name": "batchId",
+            "in": "path",
+            "type": "integer",
+            "required": true
```

Comment:
There should also be a firebase object, which give you the option to enter the token in swagger is nice to add, as it would know that you are authenticated.
`{
            "name": "authorization",
            "in": "header",
            "description": "Firebase token",
            "required": true,
            "type": "string",
            "default": "Bearer ENTER_FIREBASE_TOKEN"
          }`